chapters/ch13/1.setting-up-the-outdoors-dataset.ipynb

```diff
@@ -22,14 +22,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "sys.path.append(\"../..\")\n",
-    "import json\n",
     "import html\n",
-    "from aips import get_engine\n",
-    "from IPython.display import display, HTML\n",
+    "import json\n",
     "\n",
-    "engine = get_engine()"
+    "from IPython.display import HTML, display\n",
+    "\n",
+    "from aips import get_engine, set_engine\n",
+    "engine = get_engine(\"solr\")"
```

Comment:
Need to remove he hard-coded engine (Solr) and verify this notebook also works fine with OpenSearch.

Reply:
I've verified Opensearch's functionality for ch13 and removed this accidental hard coding